packages/nova-react/src/graphql/taggedNode.test.ts

```diff
@@ -1,7 +1,14 @@
+/**
+ * @vitest-environment node
```

Comment:
For my own curiosity and anyone reading next :) 

https://vitest.dev/guide/environment.html#environments-for-specific-files